lib/bin/run_test.js

```diff
@@ -63,6 +63,23 @@ var _ = require('lodash');
 var co = require('co');
 var exit = require('exit');  // process.exit that works on Windows
 var promiseUtil = require('../promise_util');
+var cleaned = false;
+
+function cleanup(exitCode) {
+  if (cleaned) {
+    return;
+  }
+  cleaned = true;
+  require('ps-tree')(process.pid, function (err, children) {
+    var childProcesses = children.map(function (p) { return p.PID; });
+    childProcesses.forEach(function(pid) {
+      if (require('is-running')(pid)) {
+        process.kill(pid);
```

Comment:
It might be possible and goot to write a test that caches by having a subprocess that catches and ignores SIGTERM